test/kubemark/start-kubemark.sh

```diff
@@ -425,10 +479,14 @@ done
 
 echo "DBG: TENANT_SERVER_KUBECONFIGS: ${TENANT_SERVER_KUBECONFIGS}"
 
-RESOURCE_SERVER_KUBECONFIG="/kubeconfig/rp.kubeconfig"
-echo "DBG: RESOURCE_SERVER_KUBECONFIG: ${RESOURCE_SERVER_KUBECONFIG}"
+for (( rp_num=1; rp_num<=${SCALEOUT_RP_COUNT}; rp_num++ ))
+do
+  RP_NUM=${rp_num}
+  RESOURCE_SERVER_KUBECONFIG="/kubeconfig/rp.kubeconfig-${rp_num}"
+  echo "DBG: RESOURCE_SERVER_KUBECONFIG: ${RESOURCE_SERVER_KUBECONFIG}"
 
-start-hollow-nodes
+  start-hollow-nodes
+done
```

Comment:
how scale-up to start hollow-nodes?

Reply:
the code should be clearer here with a IF ELSE block to check if it is a scaleout. env. 
it works for now is the "SCALEOUT_RP_COUNT" is set as default to 1, so for scaleup, the hollow nodes were called regardlessly.

will fix.

Reply:
fixed with a mini refactor of the code